### PR TITLE
fix: fast-layouts-react accepts null children

### PR DESCRIPTION
**Pull request by mxtopher**
# Description

This PR allows Grid to accept null children without breaking.

## Motivation & context

It is common practice in react to return null from a function or
component in order to render nothing. It is also common to use ternary
operations inside brackets to [conditionally render
elements](https://reactjs.org/docs/conditional-rendering.html)

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [X] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [X] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [X] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-layouts-react/app/index.tsx [hunk 1]
```diff
@@ -62,6 +65,26 @@ function render(): void {
                                     row={[4, 3, 3, 2]}
                                     position={[1, 1, 1, 9]}
                                 />
+                                {renderColumnA && (
```
[the remark this answers is not in the document]
Comment (mxtopher) [reply packages/fast-layouts-react/app/index.tsx]: Just for illustration purposes

File: packages/fast-layouts-react/app/index.tsx [hunk 2]
```diff
@@ -62,6 +65,26 @@ function render(): void {
...
+                                    <Column
+                                        jssStyleSheet={columnTestStyles}
+                                        span={[12, 12, 12, 4]}
+                                        row={[4, 3, 3, 2]}
+                                        position={[1, 1, 1, 9]}
+                                    >
+                                        A
+                                    </Column>
+                                )}
+                                {renderColumnB && (
```
[the remark this answers is not in the document]
Comment (mxtopher) [reply packages/fast-layouts-react/app/index.tsx]: just for illustration purposes

File: packages/fast-layouts-react/app/index.tsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ import { Pane, PaneResizeDirection } from "../src/pane";
 import { Page } from "../src/page";
 import { Grid } from "../src/grid";
 import { Column, ColumnClassNamesContract } from "../src/column";
+import { jsxAttribute } from "@babel/types";
```
Comment (nicholasrice) [inline packages/fast-layouts-react/app/index.tsx]: What is this doing?

Comment (mxtopher) [reply packages/fast-layouts-react/app/index.tsx]: This is a mistake. Removing.

**Comment by mxtopher** [comment]
Hey guys, I don't know how to get rid of this duplication warning, and I don't think this duplication is any problem, as it is just on illustration code.
Do I have to do anything?

**Comment by awentzel** [comment]
@mxtopher - I've gone ahead and approved the code duplication on codeclimate.  It should resolve itself as approved shortly.

**Pull request merged**